orc8r/cloud/go/services/tenants/servicers/protected/servicer_test.go

```diff
@@ -145,6 +148,11 @@ func TestControlProxyTenantsServicer(t *testing.T) {
 	assert.Equal(t, codes.NotFound, status.Convert(err).Code())
 	assert.Equal(t, fmt.Sprintf("controlProxy %d not found", sampleTenantID), status.Convert(err).Message())
 
+	// Get control_proxy not set
+	_, err = srv.GetControlProxyFromNetworkID(context.Background(), &tenant_protos.GetControlProxyFromNetworkIDRequest{NetworkID: "network_1"})
```

Comment:
Might be good to get the network ID from `sampleTenant` somehow instead of hardcoding.

Reply:
Done.